src/metabase/models/serialization/utils.clj

```diff
@@ -0,0 +1,48 @@
+(ns metabase.models.serialization.utils
```

Comment:
One thing that might be useful (unless I've missed it) is some explicit guidance for what to do when adding a new model. i.e. how to determine whether a model should be included in serialization, and if so, what the identity hash should be composed of.

Maybe that could go in this namespace docstring, or in a comment somewhere? We also have [this wiki page](https://github.com/metabase/metabase/wiki/Adding-a-new-model) that should probably be updated with relevant info as well.

Reply:
I've expanded the ns docstring and the `identity-hash-fields` one. I'll update the wiki page separately.